docker/release/dockercomposefiles/docker-compose-2.x.yml

```diff
@@ -35,6 +36,7 @@ services:
       - cluster.initial_cluster_manager_nodes=opensearch-node1,opensearch-node2
       - bootstrap.memory_lock=true
       - OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m
+      - OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_INITIAL_ADMIN_PASSWORD}
```

Comment:
these changes the default docker-compose files for 2.x line to now include an additional env variable (to be passed by user) to set the admin password.

This will be accompanied by the change in download instructions on the project-website ([file to change](https://github.com/opensearch-project/project-website/blob/main/_includes/downloads/opensearch-docker.markdown?plain=1)) to reflect the new requirement as an additional step.

Reply:
We need to add a comment for this line, so that people knows it is only for 2.12.0 and up.
Also, the documentation and the download page needs to be tweaked with the same changes as well.

https://opensearch.org/docs/2.11/install-and-configure/install-opensearch/docker/
https://opensearch.org/downloads.html

Reply:
Hey @DarshitChanpura please take a look at this.

Reply:
I added it above, and following the convention, I didn't add it second time as no other settings that have comments on the first one have comments here.

Reply:
@prudhvigodithi @peterzhuamazon ^^